### PR TITLE
Fix manifest image update and image tag for Keepalived

### PR DESCRIPTION
**Pull request by kashifest**
when a PR is going in BMO, keepalived image is also rebuilt locally it was not correctly updated in manifest. This PR would fix it. 

This PR also fixes a bug which was exporting keepalived image before the correct tag was determined.

### CODE REVIEW

**Review by tuminoid — APPROVED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
Should we not do a separate if statement for this?

```bash
if [ -n "${IRONIC_KEEPALIVED_LOCAL_IMAGE:-}" ]; then
  update_component_image Keepalived "${IRONIC_KEEPALIVED_LOCAL_IMAGE}"
else
  update_component_image Keepalived "${IRONIC_KEEPALIVED_IMAGE}"
fi
```
I'm thinking it would be very confusing if someone is trying to test keepalived and not BMO if we have the same check for both.

**Comment by kashifest** [comment]
> Should we not do a separate if statement for this?
> 
> ```shell
> if [ -n "${IRONIC_KEEPALIVED_LOCAL_IMAGE:-}" ]; then
>   update_component_image Keepalived "${IRONIC_KEEPALIVED_LOCAL_IMAGE}"
> else
>   update_component_image Keepalived "${IRONIC_KEEPALIVED_IMAGE}"
> fi
> ```
> 
> I'm thinking it would be very confusing if someone is trying to test keepalived and not BMO if we have the same check for both.

Good point, fixed.

**Review by lentzi90 — COMMENTED** [review]
/approve

File: lib/common.sh
Comment (kashifest) [inline lib/common.sh:259]: We don't need to export keepalived image here since after the correct tag's are set, keepalived image var is exported at line 291

**Comment by kashifest** [comment]
@lentzi90 after your approval , I have pushed a new commit. Ideally the approve label should have been removed. But it is there. I am putting a hold for you to review again. Thanks. 
/hold

File: lib/common.sh
Comment (tuminoid) [reply lib/common.sh:259]: Good find.

**Review by tuminoid — APPROVED** [review]
/lgtm

**Review by lentzi90 — COMMENTED** [review]
/approve
/hold cancel

**Pull request merged**
